int32 camera_id
TagInfos[] tag_infos
